src/Provider.js

```diff
@@ -1,16 +1,16 @@
-import React, { Component, PropTypes as T, Children } from 'react';
-import { AlgoliaSearchHelper } from 'algoliasearch-helper';
+import {Component, PropTypes, Children} from 'react';
 
-import createStore from './createStore';
-import storeShape from './storeShape';
+import createStore from './createStore.js';
+import storeShape from './storeShape.js';
 
 class Provider extends Component {
   static propTypes = {
-    helper: T.instanceOf(AlgoliaSearchHelper),
+    helper: PropTypes.object.isRequired,
```

Comment:
We have the helper available anyway in peerDependencies, so why not check its type is correct?